src/app/base/components/GlobalSideNav/_index.scss

```diff
@@ -0,0 +1,166 @@
+@mixin GlobalSideNav {
+  .l-navigation {
+    @include vf-animation('background-color');
+    width: 244px;
```

Comment:
@ndv99  You seem to be overwriting the default vanilla framework width of `15rem` which causes the "on hover" issue you mentioned.

Reply:
Ahhh that makes sense, I've just pushed a commit fixing that